src/apdu_handlers/load_metadatas.c

```diff
@@ -11,7 +11,7 @@ int load_metadatas(uint8_t p1, uint8_t p2, const buf_t *input) {
     }
     if (app_state.user_approval == false) {
         app_state.bytes_transferred = 0;
-        message_pair_t msg = {"Overwrite", "metadatas ?"};
+        message_pair_t msg = {"Overwrite", 10, "metadatas ?", 12};
```

Comment:
could be `sizeof("Overwrite") )` instead of hardcoded 10. Same at different places.

Reply:
Not `sizeof`, as it would return the size of a `char *`, but `strlen` is well enough. I don't really know why I made all this faf in the first place. I'll clean it.